lib/Cake/View/Errors/fatal_error.ctp

```diff
@@ -36,4 +36,3 @@
 if (extension_loaded('xdebug')) {
 	xdebug_print_function_stack();
 }
-?>
```

Comment:
Fixed that in https://github.com/cakephp/docs/commit/9ed77c156da6a6bce3834ec1f8f61bc93d5b9da6